app/assets/javascripts/postings.js

```diff
@@ -1,8 +1,13 @@
-$(document).on ('page:change', function() {
-
-  $( "#from-date" ).datepicker();
-  $( "#to-date" ).datepicker();
+jQuery(function($) {
+  $(document).ready( function() {
+    //enabling stickUp on the '.navbar-wrapper' class
+    $('.navbar-wrapper').stickUp();
+  });
+});
 
-  $('.navbar-wrapper').stickUp();
-
-})
+$(document).on ('page:change', function() {
```

Comment:
We don't need both $(document).on and $(document).ready.  We can just use one or the other.